src/Tizen.Uix.Tts/Tizen.Uix.Tts/TtsClient.cs

```diff
@@ -589,6 +623,51 @@ public TtsClient()
             }
         }
 
+        /// <summary>
+        /// Event to be invoked when state of TTS service changes.
```

Comment:
just a suggestion:
```suggestion
        /// Event to be invoked when the state of TTS service changes.
```

Reply:
Thank you for suggestion. I will fix it soon.